tests/RevisionTest.php

```diff
@@ -79,4 +79,93 @@ public function testRevisionsStored()
         // we should have two revisions to my name
         $this->assertCount(2, $user->revisionHistory);
     }
+
+    /**
+     * Make sure additional fields are saved with revision
+     */
+    public function testRevisionStoredAdditionalFields()
+    {
+        $this->loadMigrationsFrom([
+            '--database' => 'testing',
+            '--path' => realpath(__DIR__.'/migrations'),
+        ]);
+
+        $this->app['config']->set('revisionable.additional_fields', ['additional_field']);
+
+        $user = User::create([
+            'name' => 'James Judd',
+            'email' => 'james.judd@revisionable.test',
+            'additional_field' => 678,
```

Comment:
@joshuaziering It looks like additional fields are being added as an array above. Does this test confirm that works?